.gitignore

```diff
@@ -0,0 +1 @@
+go.work
```

Comment:
Don't we want this file in the repo?

Reply:
The tutorials suggest that each developer has their own file locally and that it's not added in the repository.

Reply:
I think we're in the "monorepo" case of https://github.com/golang/go/issues/53502 , i.e. we want to have one module for tdh2 crypto and another for the plugin, both in the same repo. So it seems that in our particular scenario we should check it in.